ota-plus-web/docs/modules/ROOT/pages/use-case-ecm.adoc

```diff
@@ -0,0 +1,287 @@
+
+Your head developer tells you that this is the software problem. To save energy, an idling engine is configured to shut down after the temperature falls below a certain level. Luckily, you can fix this issue with an OTA update.
+
+== Your production environment
+
+You have your vehicle data in a sophisticated system called "CarHQ" which integrates with your customer database called "CustomerHQ". 
+
+Your "CarHQ" system contains detailed information about all the vehicles in your fleet such as model, build date, warranty date and region. It can also tell you information about customers, what vehicle they have purchased and what support contract they're on.
+
+== A new release is announced
+
+Your release manager contacts you and says that they want to update the software on the Engine Control Module (ECM). 
+
+* The goal is to extend the engine idle time as much as possible to allow the Heating, ventilation, and air conditioning (HVAC) to keep the driver comfortable during extreme weather conditions. 
+* The update will override engine shutdown if temperature falls below 7.2 degrees Celsius or above 35 degrees Celsius.
+
+They have the following release conditions:
+
+* The release only applies to editions of the Lannista "Tireon" model built after 2018
+* Restrict the update to the Scandinavian market 
+* Target only customers who have opted for premium support.
+
+== Device groups
+
+=== Preparation
+You can run detailed search queries in CarHQ and export the data to a CSV file. Your search query might look something like this. It's simplified for the purposes of this tutorial.
+
+[source,sql]
+----
+Select VIN
+FROM CustomerView
+WHERE model="Tireon" AND "BuildDate => 2018-01-01" AND Region IN ('DK, NO, SE, FI') AND SupportLevel ="010A"
+----
+
+In this case, you'd want to create a fixed group. 
+
+* A smart group is not sophisticated enough for your query since smart groups can only filter by Device ID.
+* Instead, you can use a fixed group and import the search results that you exported from CarHQ.
+
+
+*To prepare the data for import:*
+
+. You'd export your query to a CSV file. Open the file in Excel and remove all any other columns except the VIN numbers.
+. Save a copy of this file as a TEXT file as "__Tireon-vins.txt__".
+. **Tidy up the file format**: Open the text file in a text editor such as Notepad++ and make sure there are no carriage return characters, and the lines are separated only by new line characters. 
+** In Notepad++ you can click the *Show All Characters* button to show non-printable characters such as tabs and new lines.
+** In Notepad++ you also find carriage returns by searching for the regular expression "`\r`".
```

Comment:
I think this part goes too deep into fiddly details. Since it's a use case and not a step-by-step guide, I'd prefer to just add a 3rd bullet at line 42 saying something like "To prepare the data for import, you'd export the results of your query to a text file", and optionally link to the section of the docs that describes the requirements for that text file--if it's important for us to educate our users on the finer points of line-ending characters on different platforms, it should be done in the feature documentation.

Reply:
OK  - fixed by this commit: 09f8a31c2bdeb578aef8af373fdfb5b3c70af2c4